hathor/builder/builder.py

```diff
@@ -157,6 +161,7 @@ def build(self) -> BuildArtifacts:
             checkpoints=self._checkpoints,
             capabilities=self._capabilities,
             environment_info=get_environment_info(self._cmdline, peer_id.id),
+            full_verification=self._full_verification,
```

Comment:
If you pass this parameter this way, you might pass `None` but the `HathorManager.__init__(...)` expects a bool. I think you can easily fix it by doing the following:

```python
if self._full_verification is not None:
    kwargs['full_verification'] = self._full_verification
```

Reply:
But this parameter was defined as `bool`, it could not be `None`:

```python
self._full_verification: bool = False
```

Reply:
Got it. My only consideration here would be that, if we change it to `Optional[bool]`, we can skip passing the parameter which would let `HathorManager` decide the best default value for it since it's optional. If we pass `False`, we would be overriding `HathorManager` default value.

Reply:
Ahh I see. Makes sense. Changed in https://github.com/HathorNetwork/hathor-core/pull/561/commits/d8a18d2d3759bb2f674b840618cf85063236077e, thanks!

Reply:
Great! Thanks!